app/controllers/api/v1/statuses_controller.rb

```diff
@@ -34,9 +34,13 @@ def context
   end
 
   def create
+    @thread = status_params[:in_reply_to_id].blank? ? nil : Status.find(status_params[:in_reply_to_id])
+  rescue ActiveRecord::RecordNotFound
+    render json: { error: I18n.t('errors.in_reply_not_found') }, status: 404
```

Comment:
Looks like this should be a `before_action`. Then simply rendering something would stop further execution instead of requiring this awkward `else`